base_wamas_ubl/lib/wamas/utils.py

```diff
@@ -132,3 +125,75 @@ def set_value_to_string(val, ttype, length, dp):
         bool=_set_string_bool,
     )
     return setters[ttype](val, length, dp)
+
+
+def convert_unit_code(do_convert, key, val):
+    if do_convert and key in LST_FIELD_UNIT_CODE:  # noqa: F405
```

Comment:
why do we need `do_convert`?

Reply:
because when using wamas2wamas, unitcode doesn't need to convert. But I removed `do_convert` and ref code in here